modules/droid_out/droid_vout.c

```diff
@@ -359,90 +605,6 @@ int createTexture(AndroidContext *rc)
 	return 0;
 }
 
-#ifdef GLES_FRAMEBUFFER_TEST
-
-int releaseFrameBuffer(AndroidContext *rc)
-{
-	LOG( ANDROID_LOG_DEBUG, TAG, "Android Delete FrameBuffer");
-
-	glBindFramebufferOES(GL_FRAMEBUFFER_OES, 0);
-
-	if ( rc->framebuff >= 0 )
-	{
-		glDeleteFramebuffersOES(1, &(rc->framebuff));
-		rc->framebuff = -1;
-	}
-	if ( rc->depthbuff >= 0 )
-	{
-		glDeleteRenderbuffersOES(1, &(rc->depthbuff));
-		rc->depthbuff = -1;
-	}
-}
-
-int createFrameBuffer(AndroidContext *rc)
-{
-	int backingWidth;
-	int backingHeight;
-	int res;
-
-	if ( rc->framebuff >= 0 )
-		releaseFrameBuffer(rc);
-
-	LOG( ANDROID_LOG_DEBUG, TAG, "Android Create FrameBuffer"));
-
-	glGenFramebuffersOES(1, &(rc->framebuff));
-	glBindFramebufferOES(GL_FRAMEBUFFER_OES, rc->framebuff);
-
-//	glGenRenderbuffersOES(1, &(rc->depthbuff));
-//	glBindRenderbufferOES(GL_RENDERBUFFER_OES, rc->depthbuff);
-
-//	glGetRenderbufferParameterivOES(GL_RENDERBUFFER_OES, GL_RENDERBUFFER_WIDTH_OES, &backingWidth);
-//	glGetRenderbufferParameterivOES(GL_RENDERBUFFER_OES, GL_RENDERBUFFER_HEIGHT_OES, &backingHeight);
-
-//	LOG( ANDROID_LOG_ERROR, TAG, "Android Depth Buffer Size: %dx%d\n", backingWidth, backingHeight));
-
-//    glRenderbufferStorageOES(GL_RENDERBUFFER_OES, GL_DEPTH_COMPONENT16_OES, rc->width, rc->height);
-
-//    glFramebufferRenderbufferOES(GL_FRAMEBUFFER_OES, GL_DEPTH_ATTACHMENT_OES,
-//            GL_RENDERBUFFER_OES, rc->depthbuff);
```

Comment:
I definitely do not - but to be on the safe side, I only removed what was between "GLES_FRAMEBUFFER_TEST" defs